src/Bijectors.jl

```diff
@@ -249,6 +252,11 @@ include("utils.jl")
 include("interface.jl")
 include("chainrules.jl")
 
+Base.@deprecate forward(b::AbstractBijector, x) with_logabsdet_jacobian(b, x)
+
+import Base.inv
```

Comment:
A bit annoying and slightly dangerous to import `Base.inv` - but IIRC I ran into the same problem with `@deprecate` at some point :cry:

Reply:
Yes, that behavior of `@deprecate` is annoying ... I replaced it with a manual version.

Reply:
Using

```julia
@noinline function Base.inv(b::AbstractBijector)
    Base.depwarn("`Base.inv(b::AbstractBijector)` is deprecated, use `InverseFunctions.inverse(b)` instead.", :(Base.inv))
    inverse(b)
end
```

now.